.github/workflows/test-past-releases.yml

```diff
@@ -27,21 +27,7 @@ jobs:
       fail-fast: false
       matrix:
         sct_version: [ "6.5", "6.4", "6.3", "6.2", "6.1", "6.0", "5.8", "5.7", "5.6", "5.5", "5.4", "5.3.0", "5.2.0", "5.1.0" ]
-        os: [ ubuntu-latest, macos-latest, windows-2019 ]
-        exclude:
-          # Windows support was only properly introduced in 5.7
-          - sct_version: "5.6"
-            os: windows-2019
-          - sct_version: "5.5"
-            os: windows-2019
-          - sct_version: "5.4"
-            os: windows-2019
-          - sct_version: "5.3.0"
-            os: windows-2019
-          - sct_version: "5.2.0"
-            os: windows-2019
-          - sct_version: "5.1.0"
-            os: windows-2019
```

Comment:
Note to self: This change will conflict with PR #4856. Probably merge that first.